server/test/test.helper.ts

```diff
@@ -29,6 +31,7 @@ export async function createNestAppInstance() {
 
   app = moduleRef.createNestApplication();
   app.setGlobalPrefix('api');
+  app.useGlobalFilters(new AllExceptionsFilter(moduleRef.get(Logger)));
```

Comment:
Added in order to get stack trace for internal server errors during e2e spec runs